src/index.js

```diff
@@ -1 +1,23 @@
-import _ from 'lodash';
\ No newline at end of file
+// import _ from 'lodash';
+import './style.css';
+
+const arr = [
+  { description: 'wash the dishes', completed: true, index: 0 },
+  { description: 'complete To Do list project', completed: true, index: 1 },
+  { description: 'Attend standup meeting', completed: true, index: 2 },
+];
+
+const ul = document.querySelector('.list-ul');
+
+arr.forEach((value) => {
```

Comment:
Kindly render the list items in the order of their index.

Consider the code snippet below:
```
arr.sort((a, b) => a.index - b.index).forEach((value) => {
  // Your code
}
```

Reply:
Thanks, @brytebee 🙂